smartsim/_core/control/job.py

```diff
@@ -200,17 +201,15 @@ def __init__(
         self,
         job_name: str,
         job_id: t.Optional[str],
-        entity: t.Union[SmartSimEntity, EntitySequence[SmartSimEntity], JobEntity],
+        entity: t.Union[SmartSimEntity, JobEntity],
         launcher: str,
-        is_task: bool,
     ) -> None:
         """Initialize a Job.
 
         :param job_name: Name of the job step
         :param job_id: The id associated with the job
         :param entity: The SmartSim entity(list) associated with the job
```

Comment:
I am not sure if entity(list) still applies?

Reply:
removed